app/scenes/Settings/Features.tsx

```diff
@@ -3,34 +3,94 @@ import { BeakerIcon } from "outline-icons";
 import { useState } from "react";
 import * as React from "react";
 import { useTranslation, Trans } from "react-i18next";
+import Collection from "~/models/Collection";
 import Heading from "~/components/Heading";
 import HelpText from "~/components/HelpText";
 import Scene from "~/components/Scene";
 import Switch from "~/components/Switch";
 import useCurrentTeam from "~/hooks/useCurrentTeam";
 import useStores from "~/hooks/useStores";
 import useToasts from "~/hooks/useToasts";
+import PreferredCollection from "./components/PreferredCollection";
 
 function Features() {
-  const { auth } = useStores();
+  const { auth, collections } = useStores();
   const team = useCurrentTeam();
   const { t } = useTranslation();
   const { showToast } = useToasts();
+  const [fetching, setFetching] = useState(false);
+  const [fetchError, setFetchError] = useState();
+  const [publicCollections, setPublicCollections] = useState<Collection[]>([]);
+
   const [data, setData] = useState({
     collaborativeEditing: team.collaborativeEditing,
+    preferredCollectionId: team.preferredCollectionId,
   });
 
-  const handleChange = React.useCallback(
-    async (ev: React.ChangeEvent<HTMLInputElement>) => {
-      const newData = { ...data, [ev.target.name]: ev.target.checked };
-      setData(newData);
+  React.useEffect(() => {
```

Comment:
Would prefer this option on the `Team -> Details` screen, `Features` is more for enabling/disabling of entire beta features.

Maybe should change "Details" to "General" soon 🤔

Reply:
I didn't add it on the `Details` page because it felt more like a page to view the details of the team than to use features to change the Outline default flow.

Reply:
Yea it's understandable it's not clear yet - I think adding more settings under details is okay for now